src/parser/bison_parser.y

```diff
@@ -315,6 +318,7 @@ preparable_statement:
 	|	update_statement { $$ = $1; }
 	|	drop_statement { $$ = $1; }
 	|	execute_statement { $$ = $1; }
+	|   transaction_statement { $$ = $1; }
```

Comment:
Are the transaction commands part of a preparable statement? Have a look at https://www.contrib.andrew.cmu.edu/~shadow/sql/sql1992.txt - especially <SQL transaction statement> and <SQL dynamic statement> in 12.5

Reply:
Yes (in the SQL standard), see 17.6 on page 441.
Note, our current grammar cannot be mapped directly to the standard.